ui/raidboss/data/05-shb/dungeon/heroes_gauntlet.js

```diff
@@ -0,0 +1,137 @@
+'use strict';
+
+[{
+  zoneId: ZoneId.TheHeroesGauntlet,
+  timelineFile: 'heroes_gauntlet.txt',
+  triggers: [
+    {
+      id: 'Heroes Gauntlet Spectral Dream',
+      netRegex: NetRegexes.startsUsing({ id: '4FCB', source: 'Spectral Thief' }),
+      condition: Conditions.caresAboutPhysical(),
+      response: Responses.tankBuster(),
+    },
+    {
+      id: 'Heroes Gauntlet Spectral Gust',
+      netRegex: NetRegexes.headMarker({ id: '00A9' }),
+      condition: Conditions.targetIsYou(),
+      response: Responses.spread(),
+    },
+    {
+      id: 'Heroes Gauntlet Spectral Whirlwind',
+      netRegex: NetRegexes.startsUsing({ id: '4FCC', source: 'Spectral Thief', capture: false }),
+      condition: Conditions.caresAboutAOE(),
+      response: Responses.aoe(),
+    },
+    {
+      // Spectral Thief tethers to the locations where it will attack.
+      id: 'Heroes Gauntlet Spectral Tether',
+      netRegex: NetRegexes.tether({ id: '000C', capture: false }),
+      suppressSeconds: 5,
+      infoText: {
+        en: 'Away from tether marker',
+      },
+    },
+    {
+      id: 'Heroes Gauntlet Small Zombie Tether',
+      netRegex: NetRegexes.tether({ id: '0011' }),
+      condition: Conditions.targetIsYou(),
+      infoText: {
+        en: 'Zombies away from boss',
```

Comment:
I still think this is too prescriptive.  If you move early, you have to move twice too.

Reply:
How did we want to handle this again? Just "small tether zombie", or `Responses.spread()`?

Reply:
I think it'd be fine to leave this silent, especially because you can live through being hit by multiple of these and it's hard to describe.  If you wanted to put anything, I'd say "avoid orbs" or "avoid explosions" when those spawn from the zombies?

Reply:
MM. Let's do silent then for now, and we can come back to it if people complain.